app/helpers/sessions_helper.rb

```diff
@@ -16,7 +24,20 @@ def logged_in?
 
   # Logs out the current user.
   def log_out
+    forget current_user
     session.delete(:user_id)
     @current_user = nil
   end
+
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  def forget(user)
+   user.forget
+   cookies.delete(:user_id)
+   cookies.delete(:remember_token)
```

Comment:
![2021-09-09_11-05](https://user-images.githubusercontent.com/72131384/132620732-1de807ea-8e92-4390-a41b-4b6d881d9b0a.png)
use 2 space to indent not 1